src/components/organisms/UiSimpleQuestion/UiSimpleQuestion.stories.ts

```diff
@@ -84,130 +83,54 @@ export const Basic: SimpleQuestionStoryType = {
       };
     },
     template: `<UiSimpleQuestion
-    v-bind="args"
-    v-model="current"
-    :items="items"
-  />`,
+      v-model="value"
+      v-bind="args"
+      :items="items"
+    />`,
   }),
 };
-
 Basic.parameters = {
-  chromatic: { disableSnapshot: false },
   docs: {
     source: {
       code: `<template>
   <UiSimpleQuestion
-    v-bind="args"
```

Comment:
UiSimpleQuestion has `modelValue` and `items`. You should present all props in template instead using v-bind="args"